### ISSUE TITLE
VB: System.IO.FileLoadException : Could not load file or assembly 'Newtonsoft.Json, Version=10.0.0.0

### ISSUE THREAD

**Issue by riky123456** [docusign/docusign-csharp-client#235]
Not sure if it's the right place to post this, since i got this error only in vb net and not in C#.
Basically, i get errors like "Unable to load System.IdentityModel.Tokens.Jwt Version=5.1.4.0" or "Unable to load Newtonsoft.Json Version=10.0.0.0" when creating an ApiClient object like:

Dim ApiClient as ApiClient = new ApiClient("https://demo.docusign.net/restapi")

If I try to rollback my Newtonsoft version or my Docusign dll version (tried from 4.0.4 to 3.1.3), expection moves from ApiClient creation to:

ApiClient.RequestJWTUserToken

If ApiClient creation goes fine, others methods perfectly work, i can send envelopes etc.
But the method for requesting token doesn't. If i try to install older versions of newtonsoft, exception becomes "Unable to load Microsoft.IdentityModel.Logging" or "Unable to load BouncyCastle" 

Tried many times to completely uninstall Docusign dll and its dependecies, tried many versions from the latest and still can't make it work.

I'm using Visual Basic because i'm integrating Docusign with our own company-management software.

Any help or advice would be very appreciated, thanks.

**Comment by harsharahul** [comment]
Hi @riky123456 ,

As I understand from what you have mentioned. It's working for C# but not VB? And when does ApiClient creation work fine? 
The DocuSign.eSign should work with C# and VB as its tied to .NET version, so could you please check if C# and VB programs are in same .NET.

If you have any error message, copy pasting the message here would be of great help for us to debug also.

Thank you,
-Harsha

**Comment by riky123456** [comment]
Hello @harsharahul. 
I have news. I've been testing Docusign.dll with a new console app in vb.net and everything worked fine with the RequestJWTUserToken method. So my team and me think that there is a sort of conflict between the Newtonsoft.dll version required from the Docusign dll and the Newtonsoft.dll version  used by our company-management software, of which we can't see the code because we can just "extend" it. 



The error we get is:
"System.IO.FileLoadException : Could not load file or assembly 'Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference."

We were able to solve the problem using gacutil.exe. We added Newtonsoft versions (9 and 10) to the GAC and now everything is working fine. The problem this is just a temporary fix, because our software extension can't work on a pc without gacutil.exe. 

Any ideas on how we can make Docusign (and newtosonft) dlls work with other software already using Newtonsoft?

Thanks and have a good day.
